flatsurf/geometry/categories/translation_surfaces.py

```diff
@@ -259,223 +259,6 @@ def canonicalize_mapping(self):
 
             return canonicalize_translation_surface_mapping(self)
 
-        def rel_deformation(self, deformation, local=False, limit=100):
```

Comment:
@videlec this method is a bit messy and I believe superseded by the `operator+` in libflatsurf. Are there any features in here that need to be preserved or should we eventually just use the `operator+` from libflatsurf instead?

Reply:
Probably no feature needs to be preserved.

Reply:
Ok. It's implemented now with libflatsurf backing.